.github/workflows/main.yaml

```diff
@@ -9,6 +9,7 @@ jobs:
       - uses: subosito/flutter-action@v1
         with:
           flutter-version: '2.0.6'
+      - run: flutter clean
```

Comment:
You almost never need a flutter clean run in the workflow because the jobs always start with a fresh environment. The command `flutter clean` deletes the build cache, which isn't present on a fresh environment.

Reply:
Ok, that's what I thought, but I wasn't sure, so I gave it a try :-) Thanks for confirming though, I removed the clean statements.